l10n_it_fatturapa_out_rc/views/invoice_it_template.xml

```diff
@@ -52,12 +52,15 @@
             </xpath>
             <xpath expr="//DatiRiepilogo/ImponibileImporto" position="replace">
                 <ImponibileImporto
-                t-esc="format_monetary(fpa_to_eur(get_sign(record) * tax_data['ImponibileImporto'], record), euro)"
+                t-esc="format_monetary(get_sign(record) * tax_data['ImponibileImporto'], currency)"
             />
             </xpath>
             <xpath expr="//DatiRiepilogo/Imposta" position="replace">
+<!--                <Imposta-->
+<!--                t-esc="format_monetary(get_sign(record) * tax_data['Imposta'], currency)"-->
+<!--            />-->
                 <Imposta
-                t-esc="format_monetary(get_sign(record) * tax_data['Imposta'], currency)"
+                t-esc="format_monetary(fpa_to_eur(get_sign(record) * tax_data['Imposta'], record), currency)"
```

Comment:
Se arrivati a questo punto tax_data['ImponibileImporto'] è già in euro e tax_data['Imposta'] è ancora in altra valuta, il problema direi che è altrove.

Qui a livello di template o si convertono entrambi o nessuno dei due. Convertire uno solo non ha molto senso, IMHO.